nextcord/asset.py

```diff
@@ -231,6 +231,17 @@ def _from_guild_avatar(cls, state, guild_id: int, member_id: int, avatar: str) -
             animated=animated,
         )
 
+    @classmethod
+    def _from_guild_banner(cls, state, guild_id: int, member_id: int, banner: str) -> Asset:
+        animated = banner.startswith("a_")
+        format = "gif" if animated else "webp"
```

Comment:
why webp? the rest of the library prefers png if the asset is not animated.

Reply:
Wasn't aware png was an option because Discord returns every file as a webp.